.werft/ide/integration-tests-startup.yaml

```diff
@@ -90,4 +90,7 @@ pod:
           echo "integration test failed" | werft log slice "test"
           exit 1
       fi
-      echo "integration test success" | werft log slice "test"
\ No newline at end of file
+      echo "integration test success" | werft log slice "test"
+plugins:
+  # At 00:30 every day
+  cron: "0 3 * * *"
```

Comment:
Just to confirm: Isn't it "_At 03:00 every day_"? [[1](https://crontab.guru/#0_3_*_*_*)]

Reply:
Haha, you are right! 😝  I will just remove the comment all together and people can paste into https://crontab.guru/